ebos/eclwriter.hh

```diff
@@ -370,7 +378,11 @@ public:
                                 this->summaryState(),
                                 simulator_.problem().thresholdPressure().data(),
                                 curTime, nextStepSize,
-                                EWOMS_GET_PARAM(TypeTag, bool, EclOutputDoublePrecision));
+                                EWOMS_GET_PARAM(TypeTag, bool, EclOutputDoublePrecision),
+                                isFlowsn,
+                                std::move(flowsn),
```

Comment:
I don't think this will actually move anything, since the receiving function takes the argument by const reference and not move reference (i.e. `Object&&` rather than `const Object&`).